src/index.js

```diff
@@ -45,13 +51,16 @@ import InsureePendingEnrollmentReport from "./reports/InsureePendingEnrollmentRe
 
 const ROUTE_INSUREE_FAMILIES = "insuree/families";
 const ROUTE_INSUREE_FAMILY_OVERVIEW = "insuree/families/familyOverview";
+const ROUTE_INSUREE_SUBFAMILY_OVERVIEW = "insuree/subfamilies/subFamilyOverview";
 const ROUTE_INSUREE_FAMILY = "insuree/family";
 const ROUTE_INSUREE_PROFILE = "insuree/profile";
 const ROUTE_INSUREE_INSUREES = "insuree/insurees";
 const ROUTE_INSUREE_INSUREE = "insuree/insurees/insuree";
+const ROUTE_INSUREE_SUBFAMILY = "insuree/subfamily";
+
 
 const DEFAULT_CONFIG = {
-  "translations": [{ key: "en", messages: messages_en }],
+  "translations": [{ key: "en", messages: messages_en}, {key: 'fr', messages: messages_fr}],
```

Comment:
Put the FR translation in specific language pack module and remove it from here.